src/LineBot/RichMenu/RichMenuRequest.cs

```diff
@@ -0,0 +1,128 @@
+            get => _size;
+            set
+            {
+                if (value == null)
+                    throw new InvalidOperationException("The size cannot be null.");
+
+                _size = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the rich menu should be displayed by default.
+        /// </summary>
+        [JsonProperty("selected")]
+        public bool Selected { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the rich menu. This value can be used to help manage your rich menus and is not displayed to users.
+        /// Max: 300 characters.
+        /// </summary>
+        [JsonProperty("name")]
+        public string Name
+        {
+            get => _name;
+            set
+            {
+                if (string.IsNullOrWhiteSpace(value))
+                    throw new InvalidOperationException("The name cannot be null or whitespace.");
+
+                if (value.Length > 300)
+                    throw new InvalidOperationException("The name cannot be longer than 300 characters.");
+
+                _name = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the text displayed in the chat bar.
+        /// Max: 14 characters.
+        /// </summary>
+        [JsonProperty("chatBarText")]
+        public string ChatBarText
+        {
+            get => _chatBarText;
+            set
+            {
+                if (string.IsNullOrWhiteSpace(value))
+                    throw new InvalidOperationException("The chat bar text cannot be null or whitespace.");
+
+                if (value.Length > 14)
+                    throw new InvalidOperationException("The chat bar text cannot be longer than 14 characters.");
+
+                _chatBarText = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the array of area objects which define the coordinates and size of tappable areas.
+        /// Max: 20 area objects.
+        /// </summary>
+        [JsonProperty("areas")]
+        public RichMenuArea[] Areas
+        {
+            get => _areas;
+            set
+            {
+                if (value == null)
+                    throw new InvalidOperationException("The areas cannot be null.");
+
+                if (value.Length > 20)
+                    throw new InvalidOperationException("The maximum number of areas is 20.");
+
+                _areas = value;
+            }
+        }
+
+        /// <summary>
+        /// Convert for richmenuRequest.
+        /// </summary>
+        /// <param name="richMenuRequest">richMenuRequest object.</param>
+        /// <returns>RichMenuRequest's instance.</returns>
+        public static RichMenuRequest Convert(IRichMenuRequest richMenuRequest)
+        {
+            if (richMenuRequest is RichMenuRequest request)
+            {
+                return request;
+            }
+
+            return null;
```

Comment:
We should create a new instance of the `RichMenu` class here.

Reply:
An empty instance will be okay?